tests/src/Unit/KeyHelperUnitTest.php

```diff
@@ -0,0 +1,244 @@
+<?php
+
+namespace Drupal\Tests\os2web_key\Unit;
+
+use Drupal\Core\Logger\LoggerChannelInterface;
+use Drupal\os2web_key\Exception\RuntimeException;
+use Drupal\os2web_key\Plugin\KeyType\CertificateKeyType;
+use Drupal\Tests\UnitTestCase;
+use Drupal\os2web_key\KeyHelper;
+
+/**
+ * KeyHelper unit test class.
+ */
+class KeyHelperUnitTest extends UnitTestCase {
+
+  /**
+   * The key helper.
+   *
+   * @var \Drupal\os2web_key\KeyHelper|null
+   */
+  private ?KeyHelper $keyHelper;
+
+  /**
+   * The logger.
+   *
+   * @var \Drupal\Core\Logger\LoggerChannelInterface|null
+   */
+  private ?LoggerChannelInterface $mockLogger;
+
+  /**
+   * {@inheritDoc}
+   */
+  protected function setUp(): void {
+    parent::setUp();
+
+    $this->mockLogger = $this->createMock(LoggerChannelInterface::class);
+    $this->keyHelper = new KeyHelper($this->mockLogger);
+
+  }
+
+  /**
+   * Tests creation of KeyHelper.
+   */
+  public function testKeyHelperCreation() {
+    $this->assertInstanceOf(KeyHelper::class, $this->keyHelper);
+  }
+
+  /**
+   * Tests creation of RuntimeException no SSL message.
+   */
+  public function testCreateRuntimeExceptionNoSslMessage() {
+    $this->mockLogger->expects($this->once())->method('error');
+    $message = 'test';
+    $result = $this->keyHelper->createRuntimeException($message, NULL);
+    $this->assertEquals($message, $result->getMessage());
+  }
+
+  /**
+   * Tests creation of RuntimeException no SSL message.
+   */
+  public function testCreateRuntimeExceptionWithSslMessage() {
+    $this->mockLogger->expects($this->once())->method('error');
+    $message = 'test';
+    $excepted = $message . ' (some SSL error)';
```

Comment:
Is “excepted” a typo? Do you mean “expected”?

Reply:
Fixed typo.